packages/react-devtools-core/src/backend.js

```diff
@@ -15,6 +15,7 @@ type ConnectOptions = {
   port?: number,
   resolveRNStyle?: (style: number) => ?Object,
   isAppActive?: () => boolean,
+  openFileInEditor?: () => {},
```

Comment:
This type doesn't look right; [openFileInEditor](https://github.com/facebook/react-native/blob/master/Libraries/Core/Devtools/openFileInEditor.js) has no return value. It should probably be `() => void` instead.